operations/services.md

```diff
@@ -186,9 +191,15 @@ Returns all availability blocks filtered by services, unique identifiers and oth
 | `CreatedUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 3 months | Interval in which the [Availability block](#availability-block)s were created. |
 | `UpdatedUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 3 months | Interval in which the [Availability block](#availability-block)s were updated. |
 | `CollidingUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 3 months | Interval in which the [Availability block](#availability-block)s are active. |
-| `ExternalIdentifiers` | string | optional, max 1000 items | Identifiers of [Availability block](#availability-block)s from external systems |
+| `States` | array of string [Availability block state](#availability-block-state) | optional | States the availability blocks should be in. |
+| `ExternalIdentifiers` | string | optional, max 1000 items | Identifiers of [Availability block](#availability-block)s from external systems. |
 | `Extent` | [Availability block extent](#availability-block-extent) | required | Extent of data to be returned. E.g. it is possible to specify that related service orders (for example [Reservation](reservations.md#reservation)s) are returned. |
 
+#### Availability block state
+
+* `Confirmed` - Availability can be adjusted and reservations can be added to this block.
```

Comment:
Not sure if this is the correct explanation. At least I don't fully understand. @VaclavKucera Can you pls advise here, what those two states really mean?

Reply:
Yes, this is correct 🙂 But maybe it would be clearer to say that the block deducts availability. In the future we're planning a `Inquired` state, which will not deduct availability.

Reply:
Would it be enough just add the Inquired state once released and explicitly mention that it doesn't affect the total availability.

Reply:
Yup, we don't even know when we're gonna add that state yet 👍

Reply:
Cannot we add all states and say that all other states are reserved for the future? By that we will avoid breaking changes :))

Reply:
We could, but I can't guarantee that we will actually end up adding those states, so it's up to you 😅 We're planning `Inquired` for next year and sometime down the line also maybe `Released` and `Canceled.

Reply:
Does not matter at all, but if you already know, that you are adding Inquired we need to add it, otherwise we would need to ask partners again in the future and write fallbacks, ...

@VaclavKucera What is the meaning of Released? 

@vitezp Lets put all of them, write the description and meantion that those are for future usage. The same we have for Reservation state.

Reply:
I'd go with something like this:
`Confirmed` - The block deducts availability and can have reservations assigned.
`Optional` - The block deducts availability and cannot have reservations assigned.
`Inquired` - The block does not deduct availability and cannot have reservations assigned.
`Released` - The block deducts availability, but only for reservations assigned to the block before release. Remaining availability is returned back to general availability.
`Canceled` - The block does not deduct availability and cannot have reservations assigned.